.github/workflows/ci.yml

```diff
@@ -64,6 +64,11 @@ jobs:
           python: '3.12'
           nox_session: cff
 
+        - name: Check MANIFEST.in
+          os: ubuntu-latest
+          python: '3.12'
+          nox_session: manifest
+
```

Comment:
This makes it so that `MANIFEST.in` gets checked for every PR, but we can switch it to a weekly test after #2694 is merged.